index.md

```diff
@@ -11,7 +15,7 @@ title:  JuliaOpt&#58; Optimization packages for the Julia language
 The [JuliaOpt GitHub organization](https://github.com/JuliaOpt) is home to a number of optimization-related packages written in Julia. Its purpose is to facilitate collaboration among developers of a tightly integrated set of packages for mathematical optimization.
 
 - **Overview**: [presentation](https://www.youtube.com/watch?v=7LNeR299q88) and [workshop](https://www.youtube.com/watch?v=nnL7yLMVu6c) from JuliaCon 2015
-- **Meetings**: [developers meetup 2017](/developersmeetup/index.md)
+- **Developer Workshops**: [2017](meetings/mit2017/), [2018](/meetings/bordeaux2018/)
```

Comment:
I'm okay merging and iterating on the workshop page if we hold off on linking to it publicly for a bit.